build/localnet/cluster-manager/main.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	pocketLogger "github.com/pokt-network/pocket/logger"
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/runtime/defaults"
+	"github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/k8s"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+const cliPath = "/usr/local/bin/client"
+
+var (
+	rpcUrl string
+	logger = pocketLogger.Global.CreateLoggerForModule("cluster-manager")
+)
+
+func init() {
+	rpcUrl = fmt.Sprintf("http://%s:%s", runtime.GetEnv("RPC_HOST", "v1-validator001"), defaults.DefaultRPCPort)
+}
+
+func main() {
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err.Error())
+	}
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	validatorKeysMap, err := k8s.FetchValidatorPrivateKeys(clientset)
```

Comment:
Optional NIT: As I was reading the code, I saw `kubernetes.` above and am seeing `k8s.` here.

Might be easier if rename the imports to `k8s` and `pk8s.` but not a strong stane